### ISSUE TITLE
Inventorio support

### ISSUE THREAD

**Issue by ConqAra** [CleverNucleus/PlayerEx#68]
Have the tabs be on the gui of inventorio.
Just in case I've also reported the same issue to the developer of inventorio: #

Mod: [https://www.curseforge.com/minecraft/mc-mods/inventorio](https://www.curseforge.com/minecraft/mc-mods/inventorio)

**Comment by ConqAra** [comment]
![image](https://user-images.githubusercontent.com/76233356/175765131-df747afd-317f-4fac-a62c-4b77c782971a.png)

**Comment by CleverNucleus** [comment]
Hi there,
I'll look into it. Does the hotkey to open up the attributes screen still work?
Thanks.

**Comment by ConqAra** [comment]
Yeah it does, it's still defaulted to the normal inventory though.
